htlcswitch/link.go

```diff
@@ -369,6 +376,10 @@ type channelLink struct {
 	// sub-systems with the latest set of active HTLC's on our channel.
 	htlcUpdates chan *contractcourt.ContractUpdate
 
+	// shutdownRequests is a channel that the channelLink will listen on to
+	// service shutdown requests from ShutdownIfChannelClean calls.
+	shutdownRequests chan *shutdownReq
```

Comment:
nit: `shutdownRequests` -> `shutdownRequest`

Reply:
Would like a better naming since `shutdownReq` is the type... but couldn't decide.  Since it could handle multiple requests, I decided to keep it plural, wdyt?

Reply:
`shutdownRequests` sounds like it's a list of requests thb...but yeah, it's just a nit.

Reply:
changed